src/test/java/namor/template/calculator/RPNcalculatorTest.java

```diff
@@ -1,43 +1,45 @@
 package namor.template.calculator;
 
+import namor.template.stack.ArrayStack;
+import namor.template.stack.MapBasedStack;
 import org.junit.Before;
 import org.junit.Test;
 
+import java.util.zip.DeflaterOutputStream;
+
 import static org.assertj.core.api.Assertions.assertThat;
 
 public class RPNcalculatorTest {
 
     private RPNcalculator rpnCalculator;
 
-    @Before
-    public void setUp() {
-        rpnCalculator = new RPNcalculator();
-    }
-
     @Test
     public void addTwoNumbers() {
-        final int expected = 6;
-        final int actual;
-        final String arithmeticExpression = "2 4 +";
+        final double expected = 6.4;
+        final double actual;
+        rpnCalculator = new RPNcalculator(new ArrayStack<Double>());
+        final String arithmeticExpression = "2.1 4.3 +";
 
         actual = rpnCalculator.calculate(arithmeticExpression);
         assertThat(actual).isEqualTo(expected);
     }
 
     @Test
     public void substactTwoNumbers() {
-        final int expected = 5;
-        final int actual;
-        final String arithmeticExpression = "10 5 -";
+        final double expected = 5;
+        final double actual;
+        rpnCalculator = new RPNcalculator(new ArrayStack<Float>());
+        final String arithmeticExpression = "10f 5f -";
```

Comment:
Why do you write in expression "f" pointer. How often have you seen  in calculator such approach ? it should be simple numbers without any pointers in the end like in real calculator